File: busqlineal.js
function findEle(inputArr, s, targetEle) {
    for (let j = 0; j < s; j++) {
        if (inputArr[j] === targetEle) {
            return j;
        }
    }

    return -1;
}

let inputArr = [12, 34, 10, 6, 40, 89, 57, 19, 69];
let targetElement = 40;

let s = inputArr.length;

let idx = findEle(inputArr, s, targetElement);

if (idx !== -1) {
    console.log(
        "El elemento se encuentra en la posicion: " + (idx + 1)
    );
} else {
    console.log("No se encuentra el elemento.");
}

File: busqlineal.cs
using System;

class Program
{
    static int findEle(int[] inputArr, int s, int targetEle)
    {
        for (int j = 0; j < s; j++)
        {
            if (inputArr[j] == targetEle)
            {
                return j;
            }
        }

        return -1;
    }

    static void Main()
    {
        int[] inputArr = { 12, 34, 10, 6, 40, 89, 57, 19, 69 };
        int targetElement = 40;

        int s = inputArr.Length;

        int idx = findEle(inputArr, s, targetElement);

        if (idx != -1)
        {
            Console.WriteLine(
                "El elemento se encuentra en la posicion: " + (idx + 1)
            );
        }
        else
        {
            Console.WriteLine("No se encuentra el elemento.");
        }
    }
}